### PR TITLE
fix: update design kit links

### LINKED ISSUE #252: [Design Kit Links] Broken across several pages
**From Wonil**

Update links to Design Kit resource cards on the following pages:

Pictograms
https://www.ibm.com/design/language/elements/pictograms/design

Pictograms (contribute tab)
https://www.ibm.com/design/language/elements/pictograms/contribute

Type Specs
https://www.ibm.com/design/language/elements/type-specs-ui

sketch://add-library/cloud/nwqmk

### PR DESCRIPTION
**Pull request by alisonjoseph**
Closes #252

### CODE REVIEW

File: src/pages/elements/icons/design.mdx [hunk 1]
```diff
@@ -31,7 +31,7 @@ typeface and work well at small sizes.
     <ResourceCard
       subTitle="Setup the IBM Design kit"
       aspectRatio="2:1"
-      href="https://ibm.com/standards/web/design-kitt"
+      href="sketch://add-library/cloud/nwqmkt"
       >
 
 ![](/images/SketchIcon.svg)
```
[the remark this answers is not in the document]
Comment (alisonjoseph) [reply src/pages/elements/icons/design.mdx]: I'm not sure what the fallback would be, we don't provide anything else anywhere else. 

I'll fix the image, good catch!

File: src/pages/elements/icons/design.mdx [hunk 2]
```diff
@@ -31,10 +31,10 @@ typeface and work well at small sizes.
     <ResourceCard
       subTitle="Setup the IBM Design kit"
       aspectRatio="2:1"
-      href="https://ibm.com/standards/web/design-kitt"
+      href="sketch://add-library/cloud/nwqmkt"
```
Comment (aagonzales) [inline src/pages/elements/icons/design.mdx]: Link is wrong, remove the last "t"
`sketch://add-library/cloud/nwqmk`

File: src/pages/elements/color.mdx [hunk 3]
```diff
@@ -32,7 +32,7 @@ digital world.
     <ResourceCard
       subTitle="Setup the IBM Design kit"
       aspectRatio="2:1"
-      href="https://ibm.com/standards/web/design-kitt"
+      href="sketch://add-library/cloud/nwqmkt"
```
Comment (aagonzales) [inline src/pages/elements/color.mdx]: Link is wrong, remove the last "t"
`sketch://add-library/cloud/nwqmk`

**Comment by alisonjoseph** [comment]
updated @aagonzales !

**Pull request merged**